packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_summary_card.dart

```diff
@@ -26,6 +26,7 @@ class KnowledgePanelSummaryCard extends StatelessWidget {
               cardEvaluation: getCardEvaluationFromKnowledgePanelTitleElement(
                 knowledgePanel.titleElement!,
               ),
+              showTrailingArrow: false,
```

Comment:
There might be some children in expanded panels

Reply:
Even in the "details" view?

Reply:
You can try with the Eco-Score, there are 2 levels of hierarchy (or more)

Reply:
After looking at the code, my implementation seems OK.
It's only when the item is not clickable that the arrow shouldn't be visible, right?

Reply:
Yes. I should have tested it before commenting